MapboxNavigation/RouteMapViewController.swift

```diff
@@ -388,6 +388,35 @@ class RouteMapViewController: UIViewController {
         instructionsBannerView.maneuverView.step = routeProgress.currentLegProgress.upComingStep
     }
     
+    func updateNextBanner(routeProgress: RouteProgress) {
+        guard let step = routeProgress.currentLegProgress.upComingStep,
+            routeProgress.currentLegProgress.currentStepProgress.durationRemaining <= RouteControllerHighAlertInterval,
+            let nextStep = routeProgress.currentLegProgress.stepAfter(step)
```

Comment:
This should actually be `RouteControllerHighAlertInterval * RouteControllerLinkedInstructionBufferMultiplier `